target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -222,7 +222,8 @@ case "$FIRMWARE" in
 	tplink,archer-c60-v2|\
 	tplink,archer-c60-v3|\
 	tplink,archer-c6-v2|\
-	tplink,archer-c6-v2-us)
+	tplink,archer-c6-v2-us|\
+	tplink,tl-wa1201-v2)
 		caldata_extract "art" 0x5000 0x2f20
 		ath10k_patch_mac $(macaddr_add $(mtd_get_mac_binary info 0x8) -1)
```

Comment:
Have you checked the MAC assignment with vendor firmware?

https://openwrt.org/docs/guide-developer/device-support-policies#mac_addresses

Reply:
It generates the same mac addresses for the ethernet port (there is just one) and 2g/5g.

Reply:
Then please create a table as in the referenced example and add it to the commit message. This way your check will help other people not to have to do it again.